content/exchange/artifacts/Windows.Forensics.Jumplists_JLECmd.yaml

```diff
@@ -12,8 +12,8 @@ type: CLIENT
 
 tools:
   - name: JLECmd
-    url: https://f001.backblazeb2.com/file/EricZimmermanTools/JLECmd.zip
-    expected_hash: 10bc3698867be7c707500afc01c45c44b6a0439d14e04477e08b76d95c87255c
+    url: https://download.mikestammer.com/net6/LECmd.zip
```

Comment:
Is this correct? are JLECmd.zip vs LECmd.zip the same tool?

Reply:
Sorry for my mistake. I put the wrong link. Now I fixed it. :)